### ISSUE TITLE
Broken area reference in some modal boxes

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#6100]
### What steps will reproduce the problem?

Create a report on https://wave.webaim.org/report

### What is the expected result?

No error

### What do you get instead?

2 errors about the space stream filter form.
See bellow the full error.

### Additional info

The reason is the `aria-labelledby` that should be removed when no related title.
Files concerned:
- https://github.com/humhub/humhub/blob/master/protected/humhub/modules/ui/form/widgets/views/markdownModals.php
- https://github.com/humhub/humhub/blob/master/protected/humhub/widgets/views/markdownFieldModals.php

Should I send a PR?

| Q                | A
| ---------------- | ---
| HumHub version   | 1.13.1

### Full error

**What It Means**

An aria-labelledby or aria-describedby reference exists, but the target for the reference does not exist.

**Why It Matters**

ARIA labels and descriptions will not be presented if the element referenced does not exist in the page.

**What To Do**

Ensure the element referenced in the aria-labelledby or aria-describedby attribute value is present within the page and presents a proper label or description.

**The Algorithm... in English**

An element has an aria-labelledby or aria-describedby value that does not match the id attribute value of another element in the page.

**Comment by samuk** [comment]
Following issue. Accessibility is important to us.

**Comment by funkycram** [comment]
@luke- https://github.com/humhub/humhub/pull/6177
I've removed the unused `aria-labelledby`.
Maybe there came here because of a copy/paste from https://github.com/humhub/humhub/blob/master/protected/humhub/widgets/views/markdownEditor.php

**Comment by luke-** [comment]
@funkycram Thanks is merged
